public/app/features/dashboard-scene/scene/PanelLinks.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { LinkModel } from '@grafana/data';
+import { SceneComponentProps, SceneObjectBase, SceneObjectState } from '@grafana/scenes';
+import { Dropdown, Menu, ToolbarButton } from '@grafana/ui';
+
+interface VizPanelLinksState extends SceneObjectState {
+  links?: LinkModel[];
+  menu: VizPanelLinksMenu;
+}
+
+export class VizPanelLinks extends SceneObjectBase<VizPanelLinksState> {
+  static Component = VizPanelLinksRenderer;
+}
+
+function VizPanelLinksRenderer({ model }: SceneComponentProps<VizPanelLinks>) {
+  const { menu } = model.useState();
+
+  return (
+    <Dropdown
+      overlay={() => {
+        return <menu.Component model={menu} key={menu.state.key} />;
+      }}
+    >
+      <ToolbarButton icon="external-link-alt" iconSize="md" aria-label="panel links" />
+    </Dropdown>
+  );
+}
```

Comment:
Mind that the links are always rendered in the menu. In current dashbard, when there is a single link, it's rendered just as a link, no menu.

For implementation simplicity I'm always rendering links in the menu now, as otherwise we would have to pass panel model to the scene object in order to interpolate the links via the panel links supplier.


But there's probably some refactoring of `getPanelLinksSupplier` that we can do to make it simpler. I'll try that next week to see if we can make this happen.